src/Module/Conversation/Community.php

```diff
@@ -245,8 +246,8 @@ protected function parseRequest()
 		}
 
 		self::$min_id = $_GET['min_id'] ?? null;
-		self::$max_id   = $_GET['max_id']   ?? null;
-		self::$max_id   = $_GET['last_commented'] ?? self::$max_id;
+		self::$max_id = $_GET['max_id'] ?? null;
+		self::$max_id = $_GET['last_commented'] ?? self::$max_id;
```

Comment:
This can be simplified to a single line.
```suggestion
		self::$max_id = $_GET['last_commented'] ?? $_GET['max_id'] ?? null;
```

Reply:
Done